include/RAJA/policy/openmp/policy.hpp

```diff
@@ -76,6 +76,10 @@ struct Distribute {
 //////////////////////////////////////////////////////////////////////
 //
 
+struct omp_region
+    : make_policy_pattern_t<Policy::openmp, Pattern::forall, omp::For> {
```

Comment:
This shouldn't be `Pattern::forall`.

Reply:
I can create a new pattern: Pattern::region.

Reply:
Yup, that makes sense.